### ISSUE TITLE
Using Cloudflare URL, it still polls DNS for 'dns.google.com'

### ISSUE THREAD

**Issue by vaapicon** [aarond10/https_dns_proxy#34]
Using Cloudflare URL, it still polls DNS for 'dns.google.com'. It doesn't lookup DNS for 'cloudflare-dns.com' until the I send it a request. The address it starts using for cloudflare is IPV6. The connections to the DoH server are IPV6 unless using Google's server. I thought wow, but after a few requests it stops responding. I had to restart it.    ps. Cloudflare uses 1.1.1.1 and 1.0.0.1 as Alternate names in their Certificates.

**Comment by vaapicon** [comment]
### It Works
 `Cloudflare's dns is in my cache upon startup of "https_dns_proxy" ` 
        or
  `/etc/resolv.conf is not pointing to 127.0.0.1  e.g 8.8.8.8 etc.`

### Not Working
 `Cloudflare's dns is not in my cache upon startup of "https_dns_proxy" ` 
        and
  `/etc/resolv.conf is is 127.0.0.1 `

**Comment by vaapicon** [comment]
### Output from https_dns_proxy
```
[D] 1524418344.949218 main.c:93 Received request for 'google.com' id: f261, type 28, flags 0120
[D] 1524418344.949246 https_client.c:57 Requesting HTTP/1.1: 0

[D] 1524418346.949823 https_client.c:96 CURLINFO_EFFECTIVE_URL: https://cloudflare-dns.com/dns-query?ct=application/dns-json&name=google.com&type=28
[D] 1524418346.949842 https_client.c:108 CURLINFO_RESPONSE_CODE: 0
[D] 1524418346.949845 https_client.c:169 Times: 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 2.000543
[D] 1524418346.949856 main.c:58 buflen 0
```

**Comment by aarond10** [comment]
Whoops! I'm still on holidays and kind of rushed this given the forking that was happening. Sorry!
I believe this is fixed now with commit c7979c9a3d. Please let me know if this isn't the case.

**Comment by vaapicon** [comment]
Thanks, that did the job. It works.
